client/src/components/dashboard/Preferences.js

```diff
@@ -118,31 +119,31 @@ class Preferences extends React.Component {
     }
   }
 
-  toggleMentorship = (bool, id) => {
-    var { user } = this.state;
-    if (id === 'mentorship') {
-      user.mentorship.isMentor = bool;
-    } else {
-      user.mentorship.isMentee = bool;
-    }
-    this.setState({ user });
-  }
+  /**********************
+  HANDLE CHANGE FUNCTIONS
+  **********************/
 
   handleSkillsChange = (e, { value }) => {
     var { user } = this.state;
-    user.skillsAndInterests.coreSkills = value;
+    user.skillsAndInterests.coreSkills = value
+      /* extra logic here and in sister function
+      to make sure same skills with different cases
+      are not added to the list */
```

Comment:
Having this comment break the function chaining makes it harder to follow what's happening and is somewhat unusual, I'd probably move the comment move the comment up a few lines.